fixtures/cameo/ts-100-ww.json

```diff
@@ -0,0 +1,174 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "TS 100 WW",
+  "shortName": "ts-100-ww",
+  "categories": ["Dimmer"],
+  "meta": {
+    "authors": ["Nikita"],
+    "createDate": "2024-09-03",
+    "lastModifyDate": "2024-09-03"
+  },
+  "links": {
+    "manual": [
+      "https://www.cameolight.com/en/downloads/file/id/-2008593426"
+    ],
+    "productPage": [
+      "https://www.cameolight.com/en/solutions/dj-musicians/static-lighting/theater-lights/19235/ts-100-ww"
+    ]
+  },
+  "physical": {
+    "dimensions": [320, 370, 310],
+    "weight": 7.4,
+    "power": 120,
+    "DMXconnector": "3-pin",
+    "bulb": {
+      "type": "LED",
+      "colorTemperature": 3100,
+      "lumens": 4100
+    },
+    "lens": {
+      "name": "Fresnel",
+      "degreesMinMax": [14, 38]
+    }
+  },
+  "availableChannels": {
+    "Dimmer": {
+      "fineChannelAliases": ["Dimmer fine"],
+      "capability": {
+        "type": "Intensity",
+        "brightnessStart": "0%",
+        "brightnessEnd": "100%"
+      }
+    },
+    "Dimmer 2": {
```

Comment:
This should be deleted

Reply:
Done by @p-rogalski in [`8ee2d6a`](https://github.com/OpenLightingProject/open-fixture-library/pull/4192/commits/8ee2d6a1f01c34600d734e355b3d2a7c63348726).